internal/commands/ip_access/delete_inputs.go

```diff
@@ -0,0 +1,35 @@
+package ipaccess
+
+import (
+	"fmt"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cli/user"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+)
+
+type deleteInputs struct {
+	cli.ProjectInputs
+	IPAddress string
+}
+
+func (i *deleteInputs) Resolve(profile *user.Profile, ui terminal.UI) error {
+	if err := i.ProjectInputs.Resolve(ui, profile.WorkingDirectory, false); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (i *deleteInputs) resolveAllowedIP(ui terminal.UI, allowedIPs []realm.AllowedIP) (realm.AllowedIP, error) {
+	if len(i.IPAddress) > 0 {
+		for _, allowedIP := range allowedIPs {
+			if allowedIP.IPAddress == i.IPAddress {
+				return allowedIP, nil
+			}
+		}
+		return realm.AllowedIP{}, fmt.Errorf("unable to find allowed IP: %s", i.IPAddress)
+	}
+
+	return realm.AllowedIP{}, nil
```

Comment:
if the `IPAddress` flag was never set, we'll return the empty `AllowedIP` here which is then passed to the `Delete` call (which will presumably fail).  we should either prompt the user for this information (you _do_ have a list of ip addresses they could select from), or take this moment to return an error early (rather than rely on the api call to fail)